scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,14 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+curl $FILE_HOST/boundary_huc08.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc10.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc12.sql.tar.gz | tar xz
+
+psql -f boundary_huc08.sql
```

Comment:
This approach appears to require a significant amount of RAM on the box issuing the `psql` command. We will probably need to come up with a streaming solution to avoid that overhead (a streaming approach may also help us avoid letting the tarballs touch disk).

Could we easily separate the SQL and data from these SQL files? I think then we might be able to steam the data in via `STDIN` using `COPY`. Unsure what format the geometries need to be in for that to happen. Might have to be something like WKT.

Were you able to get the import to work using these steps somewhere?